app/uploaders/profile_photo_uploader.rb

```diff
@@ -0,0 +1,52 @@
+class ProfilePhotoUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  if Rails.env.test?
+    storage :file
+  else
+    storage :fog
+  end
+  # storage :fog
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+
+  def store_dir
+    "public/testuploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn't been a file uploaded:
+  # def default_url(*args)
+  #   # For Rails 3.1+ asset pipeline compatibility:
+  #   # ActionController::Base.helpers.asset_path("fallback/" + [version_name, "default.png"].compact.join('_'))
+  #
```

Comment:
I believe this here will do the default image for you so you don't have to add it to your table! I would suggest taking the image you have and adding it locally (I'll add this to the other section too). This way you don't have to worry about the image disappearing from the interwebz